src/ripple/app/ledger/impl/LedgerToJson.cpp

```diff
@@ -157,6 +158,13 @@ fillJsonTx(
                     fill.ledger,
                     txn,
                     {txn->getTransactionID(), fill.ledger.seq(), *stMeta});
+            
+            // If applicable, insert cft issuance id
+            if (txnType == ttCFTOKEN_ISSUANCE_CREATE)
```

Comment:
Why is this needed?

Reply:
are you talking about the if-statement? just removed it.